web-components/src/components/tabs/Tabs.ts

```diff
@@ -274,7 +277,29 @@ export namespace Tabs {
       this.tabHiddenIdPositiveTabIndex = hiddenTab ? hiddenTab.id : undefined;
     }
 
+    /* Sort tabs and panes as per the user prefs saved in the localStorage */
+    private sortTabsAndPanes() {
+      if (!this.tabsOrderPrefsArray.length) {
+        return;
+      }
+      const comparator = (a: any, b: any) => {
+        const aName = a.getAttribute("name");
+        const bName = b.getAttribute("name");
+        if (this.tabsOrderPrefsArray.indexOf(aName) > this.tabsOrderPrefsArray.indexOf(bName)) {
```

Comment:
Please revisit the implementation of the comparator. You can store this entire index in a const **this.tabsOrderPrefsArray.indexOf(aName)** to avoid rewriting.

Reply:
Good catch

Reply:
addressed

Reply:
@sbudhia All comments are addressed. Take a look